### ISSUE TITLE
HTTP auth/redirect for hosted solution on Azure App Service does not prefix with HTTPS

### ISSUE THREAD

**Issue by jamaxey** [Azure-Samples/ms-identity-python-django-tutorial#5]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
>

### Any log messages given by the failure
>

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by amber-qian** [comment]
this app works fine on my local, after I deploy it to Azure, the redirect_uri doesn't have https prefix, and I received error:
https://django-aad.azurewebsites.net/auth/redirect?error=redirect_uri_mismatch&error_description=AADB2C90006: The redirect URI 'http://django-aad.azurewebsites.net/auth/redirect' provided in the request is not registered for the client id '5dbf6851-dffa-4b11-abee-2151a8e8625d'.
Correlation ID: 85fbdbfa-da14-4d26-a170-1361b296f5f4
Timestamp: 2022-06-06 23:09:22Z
&state=b88a02b0-a2fa-4618-85e3-17032bf2145b

![image](https://user-images.githubusercontent.com/80022670/172263992-356f8dea-d565-446e-8b06-3de5548ff907.png)

**Comment by NovaFly** [comment]
I also met this problem when I tried deploy it in our cloud environment, any idea with this issue?

>

**Comment by NZLostboy** [comment]
I haven't solved this yet, but this sounds like the issue https://github.com/AzureAD/microsoft-identity-web/issues/115

**Comment by NZLostboy** [comment]
Fix is here:  https://stackoverflow.com/questions/66977121/azure-ad-redirect-uri-changes-from-https-to-http

TLDR; Add these to settings.py:
```
SECURE_SSL_REDIRECT = True 

SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
```

I am not sure if the fix is something that belongs with this project or not, it works fine as long as you aren't trying to deploy it in Azure App Services or using a front facing proxy. Seems like a large number of people are getting trapped with this though
